include/uk/essentials.h

```diff
@@ -274,46 +274,6 @@ extern "C" {
 #define DEQUALIFY(t, a) ((t)(__uptr)(const volatile void *)(a))
 #endif
 
-/* The following code was taken from FreeBSD: sys/sys/cdefs.h */
```

Comment:
Now that this is moved to a different file, it may required that we remove the `Copyright of University of California` from the header of `essentials.h`. Yet it is unclear to me what is the limit of the imported code, but I suspect the remaining part starting from `UKCTASSERT` is original work. @skuenzer perhaps remembers?